### PR TITLE
Make sure lazyloaded images are loaded before printing.

### PR DESCRIPTION
**Pull request by dpellenwood**
## What does this do/fix?
Lazy loaded images were not being rendered for print. This bit of script makes sure the images get loaded before the print dialog is triggered.

### CODE REVIEW

**Review by ScottGarrison — APPROVED** [review]

File: wp-content/themes/core/components/image/js/image.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+};
+
+/**
+ * @function init
+ */
+const init = () => {
+	bindEvents();
+
+	console.info( 'SquareOne Theme: Initialized Image component.' );
+};
```
Comment (ScottGarrison) [inline wp-content/themes/core/components/image/js/image.js:31]: Should we check to make sure there's lazy load images on the page before kicking off the script?

Comment (dpellenwood) [reply wp-content/themes/core/components/image/js/image.js:31]: Good question. I considered that and opted against it for 2 reasons:

**Review by jamiepaul — APPROVED** [review]
🖨️ 💨

**Review by Vinsanity — COMMENTED** [review]
LGTM! 🎠 🍬

**Pull request merged**